src/app/AppHeader.vue

```diff
@@ -89,11 +88,11 @@
 
 <script lang="ts" setup>
 import { KUI_ICON_SIZE_30 } from '@kong/design-tokens'
+import { CogIcon, HelpIcon } from '@kong/icons'
```

Comment:
There's not really much benefit if this is the only place where these two icons are used 🤷🏼 -- in general you shouldn't globally install the icons b/c if the user never goes to a page where an icon is needed, you didn't need to import it.

Reply:
Yeah I getcha, I kinda mean generally should we install the icons globally like we do with Kongponents? I would have guessed the entire set is pretty small (or even smaller if we only install the ones we use) although maybe not? They are just a bunch of SVGs right? Do you know off the top of your head how big the full set is?

(completely different subject but its in my head as I'm writing this so gonna think out loud, I actually would like all the JS to be loaded upfront, personally I really notice the little pauses you get when navigating around the app at first, if I was a user I would prefer that I didn't see these little pauses, plus I'm not entirely sure what happens if these script loads hang or fail?)

Reply:
We shouldn’t do that because it will break tree shaking and make our initial bundle bigger.

I also don’t really see the advantage in avoiding a couple of imports. Imports are fine, they’re easy.

Reply:
> We shouldn’t do that because it will break tree shaking and make our initial bundle bigger.

Unless I'm mistaken, from looking at the build, these icons all come in in the `index.js` download anyway? So if we only globally installed the ones we use (which is what I'm suggesting), then tree-shaking doesn't come into it at all.

Even if we globally installed the lot, apparently it would add ~50ms to our initial download time. I would guess with this PR as they come in during the index.js download we probably add ~5-10ms anyway.

> I also don’t really see the advantage in avoiding a couple of imports. Imports are fine, they’re easy.

Well yeah they are easy, but no imports at all are easier. There are plenty of times I've been happily trying to use a kongponent and wondered why I can't see it only to realize "oh! I haven't imported it, silly me!". Then we added them globally, and I never have that facepalm moment anymore.

There's also the stickiness of these sorts of things just being "there", you're less likely to reach for or create "wrapper components" because its easier to just use the stuff thats available globally as part of the application as a whole.

Anyway, no action here from me, I don't want to hold up this PR with this discussion. Feel free to resolve for now.